2022-08-august/README.md

```diff
@@ -77,12 +77,20 @@ Feel free to reach out to anyone here on Slack, and post updates about your own
 
 Yes, I'm talking about Assemble. Or rather, Ian (14) and Sam (17) are:
 
-### TODO: Insert some catchy title for the hackathons grant>
+### 💸 Grants!!! For your hackathon!!! 💸
+![You get $500, Oprah meme](https://cloud-ncs2ypbyu-hack-club-bot.vercel.app/06qcy84.jpeg)
 
 
-### Fun things and trends
+At Assemble, <img src="https://ca.slack-edge.com/T0266FRGM-U0266FRGP-7f1bd1a815ac-512" width="16"> [Zach](https://zachlatta.com/) announced that [Hack Club and FIRST are offering a **✨ $500 grant ✨**](https://hackclub.com/hackathons/) to in-person high-school hackathons hosted before the end of 2022!!! All you need for the grant is a high-school team, a venue, and a custom website, and you get free tools and support too. Learn more at [hackclub.com/hackathons](https://hackclub.com/hackathons) and [APPLY HERE!](https://airtable.com/shrpYdJpAOphEWXan)
 
-TODO
+
+
+### Things to do while procrastinating on math homework:
+- Add your own Orpheus drawing (there are HUNDREDS!!): [github.com/hackclub/dinosaurs](https://github.com/hackclub/dinosaurs)
+- Grab some popcorn 🍿 and join [#movie-nights](https://hackclub.slack.com/archives/C025QEVNRPW) or share your barebones web creations 🕸 in [#freewrite](https://hackclub.slack.com/archives/C03TE1RLR7V)
+- Play around with a new demo game 🎮 and learn to build games in Sprig: [sprig.hackclub.dev](https://sprig.hackclub.dev)
```

Comment:
even though this goes to everyone in slack, lots of people reading this will be very unfamiliar with slack happenings. thinking of how we can introduce Sprig.

Reply:
ooo yes that makes sense! what about
`Play around and start making games with a beginner friendly tile-based js game engine`?

Reply:
OOO what if it was about the gallery instead? I think we'd get more interested clicks that way:
`Play games 🎮 fellow Hack Clubbers made using Sprig, a beginner-friendly game engine and make your own: [Sprig Gallery](https://sprig-gallery.hackclub.dev/)`

Reply:
perfect! thanks :D

Reply:
How's:

Play games 🎮 made by fellow Hack Clubbers (or code your own) using Sprig, a beginner-friendly game engine: [Sprig Gallery](https://sprig-gallery.hackclub.dev/)

Reply:
perfect!